File: Java script/question 27.js
// 27. Write a program to extract the file extension from a given filename.Example: Input: "document.pdf", Output: "pdf".
var input=prompt("enter input")
var len=input.length
var temp=0
var output=""
for(i=0;i<len;i++){
    if(input[i]=="."){
        temp=i;
    }
}
for(j=temp+1;j<len;j++){
    output=output+input[j]
}
console.log(output)

File: c language/question 27.c
// 27. Write a program to extract the file extension from a given filename.Example: Input: "document.pdf", Output: "pdf".
#include <stdio.h>
#include <string.h>
int main(){
    char input[50];
    printf("enter input");
    scanf("%s",&input);
    int len=strlen(input);
    char temp[50];
    char output[50];
    for(int i=0;i<len;i++){
        if(strcmp(input[i],'.')==0){
            temp[50]=i;
        }
    }
for(int j=temp;j<len;j++){
    output[50]=output+input[j];
}
printf("%s",output);
}